### PR TITLE
Format SAM Deploy Scripts, Remove Debugging

### PR DESCRIPTION
**Pull request by kolanos**
Also adds quotes around `${...}` to avoid globbing.

### CODE REVIEW

File: examples/sam/dotnet/deploy.sh [hunk 1]
```diff
@@ -9,11 +9,13 @@ sam build
 
 bucket="newrelic-example-${region}-${accountId}"
 
-aws s3 mb --region ${region} s3://${bucket}
+aws s3 mb --region "${region}" s3://"${bucket}"
```
Comment (MattWhelan) [inline examples/sam/dotnet/deploy.sh:12]: Let's put the bucket quotes around the whole string.

File: examples/sam/go/deploy.sh [hunk 2]
```diff
@@ -7,22 +7,23 @@ echo "region set to ${region}"
...
 
 bucket="newrelic-example-${region}-${accountId}"
-aws s3 mb --region ${region} s3://${bucket}
-aws s3 cp go-example.zip s3://${bucket}
-aws cloudformation deploy --region ${region} \
-  --template-file template.yaml \
-  --stack-name NewrelicExampleGo \
-  --capabilities CAPABILITY_IAM \
-  --parameter-overrides "NRAccountId=${accountId}"
+aws s3 mb --region "${region}" s3://"${bucket}"
```
Comment (MattWhelan) [inline examples/sam/go/deploy.sh:21]: Bucket quotes again.

File: examples/sam/node/deploy.sh [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 region=$2
+
 echo "region set to ${region}"
 
 sam build --use-container
 
 bucket="newrelic-example-${region}-${accountId}"
 
-aws s3 mb --region ${region} s3://${bucket}
+aws s3 mb --region "${region}" s3://"${bucket}"
```
Comment (MattWhelan) [inline examples/sam/node/deploy.sh:12]: Bucket quotes again.

File: examples/sam/python/deploy.sh [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
 region=$2
+
 echo "region set to ${region}"
 
 sam build --use-container
 
 bucket="newrelic-example-${region}-${accountId}"
 
-aws s3 mb --region ${region} s3://${bucket}
+aws s3 mb --region "${region}" s3://"${bucket}"
```
Comment (MattWhelan) [inline examples/sam/python/deploy.sh:12]: Bucket quotes again.

**Review by MattWhelan — CHANGES_REQUESTED** [review]

**Review by mrickard — APPROVED** [review]

**Pull request merged**